apps/taxes/src/modules/taxjar/taxjar-error-normalizer.ts

```diff
@@ -0,0 +1,17 @@
+import { TaxjarError } from "taxjar/dist/util/types";
+import { createLogger } from "../../lib/logger";
+import { TaxExternalError } from "../taxes/tax-error";
+
+export class TaxJarErrorNormalizer {
+  private logger = createLogger({ name: "TaxJarErrorNormalizer" });
+
+  normalize(error: unknown) {
+    if (error instanceof TaxjarError) {
+      this.logger.debug(error.stack, "TaxjarError occurred");
+
+      return new TaxExternalError(error.message);
+    }
+
+    return TaxExternalError.normalize(error);
+  }
+}
```

Comment:
Why do we make it a class, instead of a single `normalize` function?

Reply:
You know what, there is no good reason for it other than it is a convention. In this case, however, it doesn't really make sense. I changed it and added some tests.